code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -3353,31 +3353,43 @@ datum
 			fluid_g = 97
 			fluid_b = 10
 			transparency = 225
+			penetrates_skin = 1
 			var/music_given_to = null
+			var/furry = null
+			var/furrie = null
```

Comment:
rename these to actually descriptive vars please.

Reply:
Done.